src/common/options.cc

```diff
@@ -1840,6 +1840,30 @@ std::vector<Option> get_global_options() {
     .set_default(10)
     .set_description(""),
 
+    Option("osd_pool_default_mclock_res", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(1000.0)
```

Comment:
The default value Res is changed from 1000.0 to 0.0 and Wgt is changed from 500.0 to 1.0
The reason why the default Res and Wgt values of the Pool Queue are 1000 and 500 respectively refer to the default value of the Client class of the existing mClockOpClassQueue.
This is because Pool Queue is a client unit.
As we discussed earlier, (https://github.com/ceph/ceph/pull/18280) we need to distinguish the Op class and Each Client class through hirecical queue introductions.
However, since the hierarchical mClock queue has not been introduced yet, now all of the Op Class and Each Client class are handled in the pool queue.